provisioning/standalone/vscode/base/start.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+shopt -s dotglob
+
+if [ ! -f "/config/workspace/.vscode/.startup" ]; then
+    mkdir -p /config/workspace/.vscode/
+    cp -R /example_project/* /config/workspace
+    echo "[Persistent Only Feature]" > /config/workspace/.vscode/.startup
+    echo "If your CrownLabs instance is persistent, delete this file if you want to reset the workspace on next startup." >> /config/workspace/.vscode/.startup
```

Comment:
To completely reset the workspace, should delete the folder content at the beginning.

Reply:
We added an rm -r /config/workspace before copy.